roles/openshift_node/tasks/upgrade.yml

```diff
@@ -36,6 +36,9 @@
 # Restart all services
 - include_tasks: upgrade/restart.yml
 
+- include_tasks: dnsmasq_install.yml
```

Comment:
We might want to do this before upgrade/restart.yml above.

Reply:
Thanks, fixed.